### DISCUSSION TITLE
Move the redirector stuff into the utility directory?

### DISCUSSION THREAD

**Discussion by JustWhit3** [JustWhit3/osmanip#24] [Ideas]
Hi @joeletho ,

I was wondering if maybe would be better to put the redirector stuff you developed into the [utility](https://github.com/JustWhit3/osmanip/tree/main/src/utility) directory (in the respective hpp and cpp sections). What do you think about it? The fact is that this precious (and very appreciated) tool you developed is a support for the progress bars, more than a new feature, right? So since it has the same purposes of the [windows](https://github.com/JustWhit3/osmanip/blob/main/src/utility/windows.cpp) header for example, maybe placing it there would be a better idea.

I was opening this discussion to understand what do you think about this, since you are the main developer of that part and hope to find a suitable solution that fits also for you.

Cheers,
Gianluca.

**Comment by joeletho** [comment]
Hi @JustWhit3,

I think moving it to ```utility``` is perfectly reasonable considering the feature is itself a tool, like you said. Would you like me to make the appropriate changes and open a pull request or would you like to handle this on the main repo? I appreciate your approval of the redirection feature and for taking my opinions into consideration for potential solutions.

Thanks, Gianluca.

Joel

**Reply by JustWhit3** [reply]
Hi @joeletho it's a pleasure for me to discuss with you about consistent changes, in order to create a strong community which can take decisions all together.

However, do you prefer to apply changes on your own and send a pull request? Since you are the author of this part I would let you 100% decisions about it. Let me know, otherwise I can do it quickly also now.

**Reply by joeletho** [reply]
Hi @JustWhit3,

You have certainly been very welcoming and I've enjoyed working with you.

I have made the appropriate changes to the repo and opened a pull request. I've there is anything amiss, please let me know.

Best,
Joel
